README.md

```diff
@@ -800,6 +801,16 @@ onSessionRenewed = (
 ) => (() => void)
 ```
 
+### logout
+
+Method that requests a user logout.
```

Comment:
Do I have to add here some information about this only been available for B2P for now? The handler for this message won't be implemented on Novum AFAIK, at least not in this moment